src/command_modules/azure-cli-advisor/azure/cli/command_modules/advisor/_help.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+helps['advisor'] = """
+    type: group
+    short-summary: (PREVIEW) Manage Azure Advisor.
+"""
+
+helps['advisor configuration get'] = """
+    type: group
```

Comment:
The type is wrong.

Reply:
Same with the ones below. `type: command`

Reply:
This causes the CLI to case on `--help`.
i.e.:
```
$ az advisor configuration get -h

Group
    az advisor configuration get: Get Azure Advisor configuration.

'CommandHelpFile' object has no attribute 'children'
```